packages/flutter_test/test/widget_tester_test.dart

```diff
@@ -137,4 +137,26 @@ void main() {
       expect(failure.message, contains('Actual: ?:<zero widgets with $customDescription'));
     });
   });
+
+  group('find.descendant', () {
+    testWidgets('fails with a descriptive message', (WidgetTester tester) async {
+      await tester.pumpWidget(new Row(children: [
+        new Column(children: [new Text('foo')]),
+        new Text('bar')
+      ]));
+
+      TestFailure failure;
+      try {
+        expect(find.descendant(ancestor: find.widgetWithText(Column, 'foo'),
+                descendant: find.text('bar')), findsOneWidget);
+      } catch (e) {
+        failure = e;
+      }
+
+      expect(failure, isNotNull);
+      expect(
```

Comment:
please wrap this using flutter style (so if there's a newline after the (, there's a newline after the matching ), etc).